### PR TITLE
Cafe snacks

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Multiple snack foods I made while doing the cafe sprites a while ago, flan, cookies, cakes, and a bit more fun stuff

### CODE REVIEW

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -1313,6 +1313,95 @@
...
+	result = /obj/item/weapon/reagent_containers/food/snacks/muffin/bluespace
+
+/datum/recipe/yellowcake
+	reagents = list("uranium" = 5, "radium" = 10)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		)
+	result = /obj/item/weapon/reagent_containers/food/snacks/yellowcake
```
Comment (ComicIronic) [inline code/modules/food/recipes_microwave.dm]: If this is made in the same way as the other cakes, why isn't it a cake?

File: code/modules/food/recipes_microwave.dm [hunk 2]
```diff
@@ -1313,6 +1313,95 @@
...
+
+/datum/recipe/cookiebowl
+	reagents = list("flour" = 5, "sugar" = 2)
+	items = list (
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/chocolatebar
+		)
+	result = /obj/item/weapon/reagent_containers/food/snacks/cookiebowl
+
+/datum/recipe/chococherryecake
```
Comment (ComicIronic) [inline code/modules/food/recipes_microwave.dm]: Typo here.

File: code/modules/food/recipes_microwave.dm [hunk 3]
```diff
@@ -1313,6 +1313,95 @@
...
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/chocolatebar,
+		/obj/item/weapon/reagent_containers/food/snacks/chocolatebar,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/cherries,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/cherries
+		)
+	result = /obj/item/weapon/reagent_containers/food/snacks/sliceable/chococherrycake
+
+/datum/recipe/pumpkinbread
+	items = list(
```
Comment (ComicIronic) [inline code/modules/food/recipes_microwave.dm]: Not to be an ass, but most bread variants are cooked from flour in the microwave, not with bread slices themselves.

File: code/modules/food/recipes_microwave.dm (hunk 1 above)
Comment (NigglyWiggly) [inline code/modules/food/recipes_microwave.dm]: It's not sliceable like the other cakes, so I figured it wouldn't go with them

Comment (clusterfack) [inline code/modules/food/recipes_microwave.dm]: If thats your only problem why not make a sliceable variable

Comment (ComicIronic) [inline code/modules/food/recipes_microwave.dm]: No, that's a fair reason. A few cakes aren't sliceable.

**Comment by NigglyWiggly** [comment]
f u c k  i t

**Pull request closed without merging**